hledger-lib/Hledger/Reports/MultiBalanceReport.hs

```diff
@@ -241,7 +238,7 @@ makeReportQuery ropts reportspan
     dateqcons        = if date2_ ropts then Date2 else Date
 
 -- | Make a valuation function for valuating MixedAmounts and a given Day
-makeValuation :: ReportOpts -> Journal -> PriceOracle -> Valuation
+makeValuation :: ReportOpts -> Journal -> PriceOracle -> Day -> MixedAmount -> MixedAmount
```

Comment:
Maybe parenthesise (Day -> MixedAmount -> MixedAmount) here too, like the others below.